### PR TITLE
notify user when error is raised when evaluating breakpoint condition

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Events.java [hunk 1]
```diff
@@ -194,4 +194,22 @@ public HotCodeReplaceEvent(ChangeType changeType, String message) {
             this.message = message;
         }
     }
+
+    public static class UserNotificationEvent extends DebugEvent {
+        public enum NotifyType {
+            ERROR, WARNING
```
Comment (yaohaizh) [inline com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Events.java]: For notification type, there should be information only type.

File: com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Events.java [hunk 2]
```diff
@@ -194,4 +194,22 @@ public HotCodeReplaceEvent(ChangeType changeType, String message) {
             this.message = message;
         }
     }
+
+    public static class UserNotificationEvent extends DebugEvent {
+        public enum NotifyType {
+            ERROR, WARNING, INFORMATION
+        }
+
+        public NotifyType notifyType;
```
Comment (yaohaizh) [inline com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Events.java]: NotificationType

Comment (andxu) [reply com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Events.java]: will submit later

Comment (andxu) [reply com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Events.java]: changed at https://github.com/Microsoft/java-debug/pull/156

**Comment by andxu** [comment]
since andy_dev4 is merged to master, close this one and create a new one instead

**Comment by andxu** [comment]
using this one, since the andy_cond3 has conflicts with master https://github.com/Microsoft/java-debug/pull/158

**Pull request closed without merging**